### PR TITLE
Achiemevements page

### LINKED ISSUE #42: Achievements (model, view, imagens)
- [x] List of achievements with milestones
- [x] Achievements icons
- [x] Achievements table
- [x] Achievements being dynamically displayed in user's profile (Django View)

### PR DESCRIPTION
**Pull request by ferfahl**
From the issue #42 
Still needs to connect with user stats when it's done

### CODE REVIEW

File: media/profile_pictures/60c49d4b3a38419980d22e062cd1ffb6.png
Comment (0xEDU) [inline media/profile_pictures/60c49d4b3a38419980d22e062cd1ffb6.png:1]: I think this shouldn't be here hehe

File: static/css/components/user.css [hunk 1]
```diff
@@ -46,3 +46,11 @@ p#friendsText {
 	color: #FCFCFC;
 	font-size: 2em;
 }
+
+img#achieveImage {
```
Comment (0xEDU) [inline static/css/components/user.css]: Empty CSS can be removed

File: media/profile_pictures/60c49d4b3a38419980d22e062cd1ffb6.png
Comment (ferfahl) [reply media/profile_pictures/60c49d4b3a38419980d22e062cd1ffb6.png:1]: Removed

File: static/css/components/user.css (hunk 1 above)
Comment (ferfahl) [reply static/css/components/user.css]: Won't it break the code below?

Comment (0xEDU) [reply static/css/components/user.css]: Nope, the style will work just for the hover specifically, no need to select the whole element if it won't be styled

**Review by rodsmade — APPROVED** [review]
Tried and tested.

Looking so coooooooooool!!

**Review by 0xEDU — APPROVED** [review]
The achievements are so cuuute 🥹

**Pull request merged**